pkg/collections/offledger/dcas/cas.go

```diff
@@ -7,80 +7,71 @@ SPDX-License-Identifier: Apache-2.0
 package dcas
 
 import (
-	"crypto"
-	"encoding/base64"
-	"encoding/json"
+	"fmt"
 
-	"github.com/btcsuite/btcutil/base58"
+	"github.com/ipfs/go-cid"
+	dshelp "github.com/ipfs/go-ipfs-ds-help"
 )
 
-// GetCASKeyAndValue first normalizes the content (i.e. if the content is a JSON doc then the fields
-// are marshaled in a deterministic order) and returns the content-addressable key
-// (encoded in base64) along with the normalized value.
-func GetCASKeyAndValue(content []byte) (string, []byte, error) {
-	bytes, err := getNormalizedContent(content)
-	if err != nil {
-		return "", nil, err
-	}
-	return string(getCASKey(bytes)), bytes, nil
-}
+const (
+	dsPrefix = "/blocks"
```

Comment:
probably could use a comment on why /blocks

Reply:
OK, I'll add it in a separate PR.